lesson5_HT3/task3_1.js

```diff
@@ -0,0 +1,4 @@
+const number = 5
+for(let i = 1; i <= 10; i++){
+    console.log(`5 x ${i} = ${number*i}`)
```

Comment:
If you change the `number` variable value you should change also string. You can avoid it using interpolation 
```js
console.log(`${number} x ${i} = ${number*i}`)
```